common/config/beachball/changelog-config.ts

```diff
@@ -27,9 +27,7 @@ export const config: BeachballConfig = {
       name: "@azure/communication-react and its packlets",
       include: "packages/*",
       disallowedChangeTypes: [
-        "major",
-        "minor",
-        "patch"
+        "major"
```

Comment:
We should also disallow `prerelease`?

Reply:
Mentioned this in the PR description, I'm leaving it in to see if we should it for beta stuff